zipline/examples/__init__.py

```diff
@@ -77,6 +78,8 @@ def run_example(example_name, environ):
         analyze=getattr(mod, 'analyze', None),
         bundle='test',
         environ=environ,
+        # example test data use the VolumeShareSlippage model
+        equity_slippage=VolumeShareSlippage(),
```

Comment:
Setting this here makes the examples a little bit confusing; if you ran the example code exactly as defined in the example file, you'd get a different result than what we produce here.  It looks like we call `set_commission` in most of the examples for this reason.